docs/compression_algorithms/Pruning.md

```diff
@@ -203,3 +163,71 @@ it does not take into account the number of filters in layers that cannot be pru
 It is important to note that pruning levels mentioned in the `statistics of the filter pruning algorithm` are the goals the algorithm aims to achieve.
 It is not always possible to achieve these levels of pruning due to cross-layer and inference constraints. 
 Therefore, it is expected that these numbers may differ from the calculated statistics in the `statistics of the pruned model` section.
+
+**Example configuration files**
+
+>_For the full list of the algorithm configuration parameters via config file, see the corresponding section in the [NNCF config schema](FIXME)_.
+
+- Prune a model with default parameters (from 0 to 0.5 pruning level across 100 epochs with exponential schedule)
+```json5
+{
+    "input_info": { "sample_size": [1, 3, 224, 224] },
+    "compression":
+    {
+        "algorithm": "filter_pruning",
+    }
+}
+```
+
+- Prune a model, immediately setting pruning level to 10%, applying [batchnorm adaptation](./BatchnormAdaptation.md) and reaching 60% within 20 epochs using exponential schedule, enabling pruning of first convolutional layers and downsampling convolutional layers:
+```json5
+{
+    "input_info": { "sample_size": [1, 3, 224, 224] },
+    "compression":
+    {
+        "algorithm": "filter_pruning",
+        "pruning_init": 0.1,
+        "params": {
+            "pruning_target": 0.6,
+            "pruning_steps": 20,
+            "schedule": "exponential",
+            "prune_first_conv": true,
+            "prune_downsample_convs": true,
+        }
+    }
+}
+```
+
+- Prune a model using geometric median filter importance and reaching 30% within 10 epochs using exponential schedule, postponing application of pruning for 10 epochs:
+```json5
+{
+    "input_info": { "sample_size": [1, 3, 224, 224] },
+    "compression":
+    {
+        "algorithm": "filter_pruning",
+        "params": {
+            "filter_importance": "geometric_median",
+            "pruning_target": 0.3,
+            "pruning_steps": 10,
+            "schedule": "exponential",
+            "num_init_steps": 10
+        }
+    }
+}
+```
+
+- Prune and quantize a model at the same time using default parameters:
+```json5
+{
+    "input_info": { "sample_size": [1, 3, 224, 224] },
+    "compression":
+    [
+        {
+            "algorithm": "filter_pruning",
+        },
+        {
+            "algorithm": "quantization",
+        }
+    ]
+}
+```
```

Comment:
@daniil-lyakhov please help extend this section with easy-to-understand, common use cases and options

Reply:
@vshampor, examples looks good, only things to pay attention to is Legr algo. Baseline scheduler isn't covered here, should we add an example with baseline scheduler as well?

Reply:
Besides simple examples with `pruning_flops_target` and `all_weights` are needed

Reply:
Prune a model, immediately setting FLOPs pruning level to 10%, applying [batchnorm adaptation](./BatchnormAdaptation.md) and reaching 60% FLOPs pruning level within 20 epochs using exponential schedule
```
{
    "input_info": { "sample_size": [1, 3, 224, 224] },
    "compression": 
    [
        {           
            "algorithm": "filter_pruning",
            "pruning_init": 0.1,
            "params": 
            {
                "pruning_flops_target": 0.6,
                "pruning_steps": 20
            }
        }
    ]
}
```

Reply:
Prune a model with default parameters but all filter norms will be collected together and sorted to choose the least important ones
```json5
{
    "input_info": { "sample_size": [1, 3, 224, 224] },
    "compression":
    {
        "algorithm": "filter_pruning",
        "all_weights": true
    }
}
```

Reply:
Consolidated some examples into the existing ones and added the rest. Thanks!